salt/metalk8s/kubeadm/init/certs/ca.sls

```diff
@@ -38,8 +33,7 @@ Generate CA certificate:
     - require:
       - x509: Create CA private key
 
-# TODO: Find a better way to advetise CA server
-Advertise CA in the mine:
+Advertise CA certificate in the mine:
```

Comment:
This one (and the other CA) should be a `mine_function` in the pillar automaticaly updated with `mine.update` because if we full the mine we will need to advertise again to the mine manually.

But could be done in another PR/ticket btw

Reply:
That should be another ticket indeed.

Reply:
Created #960